src/www/app/outline_server_repository/shadowsocks_session_config.ts

```diff
@@ -13,10 +13,10 @@
 // limitations under the License.
 
 // Represents a Shadowsocks proxy server configuration.
-export interface ShadowsocksConfig {
+export interface ShadowsocksSessionConfig {
   host?: string;
   port?: number;
   password?: string;
   method?: string;
-  name?: string;
+  tag?: string;
```

Comment:
Can you remove the name, actually. We don't need it to establish a Shadowsocks session.

Reply:
(You meant the tag)